pkg/loop/internal/relayer/pluginprovider/chainreader/chain_reader_test.go

```diff
@@ -340,16 +352,25 @@ type eventConfidencePair struct {
 
 type fakeChainReader struct {
 	fakeTypeProvider
-	vals     []valConfidencePair
-	triggers []eventConfidencePair
-	stored   []TestStruct
+	vals        []valConfidencePair
+	triggers    []eventConfidencePair
+	stored      []TestStruct
 	batchStored BatchCallEntry
-	lock     sync.Mutex
+	lock        sync.Mutex
+	isStarted   bool
```

Comment:
this flag will be accessed within the methods to check if the service `isStarted`

Reply:
If it was possible for a method to be called before `Start`, then it must also be possible for a method to be called _during_ `Start`, so doesn't this field need to be safe for concurrent use?

Reply:
Thanks! addressed here: [`215fb70` (#705)](https://github.com/smartcontractkit/chainlink-common/pull/705/commits/215fb70b764ba3ff8fd0eee82682291ffb4cac5e)